index.html

```diff
@@ -48,32 +104,109 @@
+          <label>Dice Rolls:</label>
+          <!-- The self-closing <input /> tag represents a field for a user to enter input. -->
+          <input id="number-of-rolls" type="number" value="2" min="2" />
+        </div>
+        <div class="parameter">
+          <label for="game-mode">Game Mode:</label>
+          <select id="game-mode">
+            <option value="highest">Highest</option>
+            <option value="lowest">Lowest</option>
+          </select>
+        </div>
+        <div id="buttons">
+          <button id="begin-button">Begin</button>
+          <button id="remake-button">Remake</button>
+        </div>
+      </div>
+      <!-- The self-closing <br /> tag represents a line break - a line's worth of spacing before dispalying the next element. -->
       <br />
-      <button id="submit-button">Submit</button>
+      <!-- The <button> tag represents.. you guessed it! a button. -->
+      <button id="proceed-button">Proceed</button>
       <p>Output:</p>
       <div id="output-div"></div>
     </div>
-    <!-- Import program logic -->
+    <div id="leaderboard">
+      <p>========== 🏆 <b>Leaderboard</b> 🏆 ==========</p>
+      <div id="leaderboard-scores"></div>
+    </div>
+    <!-- Everything below this is incorporating JavaScript programming logic into the webpage. -->
+    <!-- The script tag encloses JavaScript syntax for the browser to interpret programtically -->
+    <!-- The following line imports all code written in the script.js file -->
     <script src="script.js"></script>
     <!-- Define button click functionality -->
     <script>
-      var button = document.querySelector("#submit-button");
-      button.addEventListener("click", function () {
-        // Set result to input value
-        var input = document.querySelector("#input-field");
-        var result = main(input.value);
+      // Declare and define the 3 parameter inputs.
+      var numPlayers = document.querySelector("#number-of-players");
+      var numRolls = document.querySelector("#number-of-rolls");
+      var mode = document.querySelector("#game-mode");
+
+      // Declare and define the buttons.
+      var btnProceed = document.querySelector("#proceed-button");
+      // "Proceed" is disabled unable player clicks "Begin".
+      btnProceed.disabled = true;
+      var btnBegin = document.querySelector("#begin-button");
+      var btnRemake = document.querySelector("#remake-button");
 
+      var output = document.querySelector("#output-div");
+      var leaderboard = document.querySelector("#leaderboard-scores");
+
+      btnProceed.addEventListener("click", function () {
+        // Store the output of main() in a new variable
+        var result = main();
+
+        // Display result in output element
+        output.innerHTML = result;
+      });
+
+      btnBegin.addEventListener("click", function () {
+        var result = beginGame(numPlayers.value, numRolls.value, mode.value);
+        console.log(mode.value);
```

Comment:
Great job playing around with the HTML file and adding event listeners, querySelectors etc!